src/app/shared/components/archive-type-select/archive-type-select.component.html

```diff
@@ -0,0 +1,28 @@
+<!-- format -->
+<div class="archive-type-select">
+  <div class="archive-image">
+    <img
+      *ngIf="selectedValue"
+      [src]="icons[selectedValue.split('+')[0]]"
+      [alt]="altTexts[selectedValue.split('+')[0]]"
+    />
+  </div>
+  <div class="archive-dropdown">
+    <select
```

Comment:
Can you make this select match the style of other dropdowns in the application? I think it might need the `form-control` class or a related class? This one just looks a bit smaller than the usual input style.